netjsonconfig/backends/zerotier/schema.py

```diff
@@ -0,0 +1,237 @@
+"""
+ZeroTier specific JSON-Schema definition
+"""
+
+from copy import deepcopy
+
+from ...schema import schema as default_schema
+
+# The schema is taken from OpenAPI specification:
+# https://docs.zerotier.com/service/v1/ (self-hosted controllers)
+# https://docs.zerotier.com/openapi/centralv1.json (central controllers)
+base_zerotier_schema = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "additionalProperties": True,
+    "properties": {
+        "zerotier": {
+            "type": "array",
+            "title": "ZeroTier",
+            "uniqueItems": True,
+            "additionalItems": True,
+            "propertyOrder": 12,
+            "items": {
+                "type": "object",
+                "title": "ZeroTier Network",
+                "uniqueItems": True,
+                "additionalProperties": True,
+                "required": ["name"],
+                "properties": {
+                    # Read-only properties
+                    "name": {
+                        "type": "string",
+                        # Since it is intended to be set by
+                        # the VPN backend's name field, it is read-only
+                        "readOnly": True,
+                        "propertyOrder": 1,
+                        "example": "openwisp-wifi-network",
```

Comment:
what is this example thing?

Reply:
I think this property was copied from the OpenAPI schema. I'll remove it. :+1: